pkg/systemstatsmonitor/README.md

```diff
@@ -40,7 +40,7 @@ Below metrics are collected from `disk` component:
 * `disk_operation_bytes_count`: # of Bytes used for reads/writes on this device
 * `disk_operation_time`: [# of milliseconds spent reading/writing][iostat doc]
 * `disk_bytes_used`: Disk usage in Bytes. The usage state is reported under the `state` metric label (e.g. `used`, `free`). Summing values of all states yields the disk size.
-FSType and MountOptions are also reported as additional information.
+* `disk_bytes_total`: Total disk space in Bytes. Additional information such as fstypes and mountoptions is also reported
```

Comment:
Why do we need this metric? I believe `disk_bytes_used[state='used']+disk_bytes_used[state='free']` should yield the result, right?
I believe the general principle is that the instrumentation should report the most fundamental & raw & duplication-free data, and we can use backend (Prometheus, Stackdriver, etc) to do more processing and piece them togerther.

Reply:
done